plugins/bkpr/bookkeeper.c

```diff
@@ -553,7 +553,6 @@ static bool new_missed_channel_account(struct command *cmd,
 				       const char *currency,
 				       u64 timestamp)
 {
-	struct chain_event *chain_ev;
 	size_t i, j;
```

Comment:
Not a big fan of this commit. Seems like you need separate "channel proposed" and "channel confirmed" events, and you're shortcutting?

Reply:
I agree. @ddustin heard me complaining about this last week... should have just done it then TM.

Reply:
Ok, fixed up so that the event log always rolls forward.